ext/opcache/jit/zend_jit_ir.c

```diff
@@ -14476,7 +14476,7 @@ static int zend_jit_fetch_obj(zend_jit_ctx         *jit,
 		ZEND_ASSERT(end_inputs == IR_UNUSED);
 		if ((res_info & MAY_BE_GUARD) && JIT_G(current_frame)) {
 			uint8_t type = concrete_type(res_info);
-			uint32_t flags = 0;
+			uint32_t flags = ZEND_JIT_EXIT_CHECK_EXCEPTION;
```

Comment:
I can't comment on the line, but maybe it's better to change line 14484 to `flags |= ZEND_JIT_EXIT_FREE_OP1;`. It doesn't matter here because you also check exceptions when OP1 needs to be freed, but it would be more "future-proof"/"semantically right".

Reply:
Oh right, that's actually a mistake. Thank you for spotting this.